README.Rmd

```diff
@@ -72,6 +72,23 @@ You can install the development version from [GitHub](https://github.com/) with:
 devtools::install_github("trytoolchest/toolchest-client-r")
 library(toolchest)
 ```
+
+### Python requirements
+The R client requires a version of **Python 3.6 or greater** with 
+`libpython` libraries, accessible via the `$PATH`. 
```

Comment:
I doubt most of the folks using R will know with `libpython` is. Maybe "with development libraries (e.g. `libpython`)"?

Reply:
This comment still holds